pkg/service/generateConfig/generateConfig.go

```diff
@@ -44,55 +44,45 @@ test:
   containerName: ""
   networkName: ""
   testSets: []
-  globalNoise: |-
-    {
-      "global": {
-        "body": {},
-        "header": {}
-      },
-      "test-sets": {
-        "test-set-name": {
-          "body": {},
-          "header": {}
-        }
-      }
-    }
+  globalNoise:
+    global:
+      body: {}
+      header: {}
+    test-sets:
+      test-set-name:
```

Comment:
I think we shouldn't give this test-set-name in the default config because people will just run it and keploy thinks test-set-name is name of the test set and it will search for test-set-name. Just write test-sets:  and  leave it . Above the globalNoise: write a comment that to use global noise  please follow the guide at the end of the yaml.